core/model/objects/message.hpp

```diff
@@ -14,12 +15,26 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#ifndef CORE_DOMAIN_TRANSACTIONS_DOMAINDEFINITIONTRANSACTION_HPP_
-#define CORE_DOMAIN_TRANSACTIONS_DOMAINDEFINITIONTRANSACTION_HPP_
+#ifndef CORE_DOMAIN_OBJECTS_MESSAGE_HPP_
+#define CORE_DOMAIN_OBJECTS_MESSAGE_HPP_
 
-#include "abstract_transaction.hpp"
+#include "object.hpp"
+#include "../../service/json_parse.hpp"
+#include <string>
 
-namespace domain_definition_transaction {
+namespace message {
+
+class Message : public Object{
+    std::string text;
+
+public:
+    Message(std::string text);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class 'Message' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/file/4345543353/issues/source?bid=3712491&fileBranchId=3870389#l31)